layouts/partials/survey.html

```diff
@@ -0,0 +1,9 @@
+<div class="survey-banner" id="survey-banner">
+    <div class="content">
+        <button type="button" id="survey-dismiss"><span>Close</span></button>
+        <p>
+            Love IRL? Take this 5 minute survey to help us make it even better.
```

Comment:
This is totally a copy thing and outside my code review perview...

I'd suggest changing it to "Love IRL? You should take this..." It i[ncreases click through rates](https://kottke.org/09/07/you-should-follow-me-on-twitter).

Reply:
I'm good with this.